apps/system/style/software_button_manager/software_button_manager.css

```diff
@@ -8,22 +8,15 @@
   left: 0;
   position: absolute;
   height: var(--software-home-button-height);
-  transition-duration: var(--transition-duration);
-  transition-timing-function: ease-in-out;
   visibility: hidden;
   background: black;
 }
 
-#screen.locked #software-buttons,
-#software-buttons.no-transition {
-  transition: none;
-}
-
 #screen:not(.locked) #software-buttons.visible,
 #screen.locked.attention #software-buttons.visible,
 #screen.locked.secure-app #software-buttons.visible {
-  transform: translateY(-100%);
   visibility: visible;
+  bottom: 0px;
```

Comment:
We can probably just get rid of the `bottom: calc(var(--software-home-button-height) * -1);` rule above and make `bottom: 0` the default since we are not animating anymore.